src/models/searchservice/response/directanswer/fieldvaluedirectanswers/AddressDirectAnswer.ts

```diff
@@ -0,0 +1,29 @@
+import { BuiltInFieldType } from '../BuiltInFieldType';
+import { BaseFieldValueDirectAnswer } from '../BaseFieldValueDirectAnswer';
+import { EnumOrLiteral } from '../../../../utils/EnumOrLiteral';
+
+/**
+ * A {@link BaseFieldValueDirectAnswer} interface with 'address' field type.
+ *
+ * @public
+ */
+export interface AddressDirectAnswer extends BaseFieldValueDirectAnswer<Address> {
+  fieldType: EnumOrLiteral<BuiltInFieldType.Address>
+}
+
+/**
+ * An interface with address fields to use in {@link BaseFieldValueDirectAnswer.value}.
+ *
+ * @public
+ */
+export interface Address {
+  line1: string,
+  line2?: string,
+  line3?: string,
+  sublocality?: string,
+  city: string,
+  region: string,
+  postalCode: string,
```

Comment:
I'm not certain if all [non US areas](https://en.wikipedia.org/wiki/List_of_postal_codes) would have postal code? I added a location for Bolivia to the account test.

Reply:
<img width="675" alt="Screen Shot 2022-08-16 at 12 58 09 PM" src="https://user-images.githubusercontent.com/36055303/184936761-5fc48641-0c32-4a5d-87f9-35c39b88a616.png">
Alright, looks like region and postalCode is not certain. let's make them optional.

Reply:
If we're not sure about things and composeAbstractSchema says they're all optional I don't mind just going with that

Reply:
I would assume at least line1 should always be defined for an address? can't say the same for the other ones though.

Reply:
Julianne talked to KM channel. Looks like countryCode should always be defined in practice and everything else is optional. So we should update to reflect that